homeTask_3/homeTask_3/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace homeTask_3
+{
+    class Program
+    {
+        //Задание- создать три массива по 10 элементов, первый
+        //заполнить значениями индексов элементов, второй - квадратами индексов,
+        //третий-кубами, и вывести на экран средние значения элементов для всех массивов
+
+        public void InitializeArray(int[] array, int n)
+        {
+            for (int i = 0; i < array.Length; i++)
+                array[i] = (int)Math.Pow(i+1, n);
+        }
+
+        public double AverageSumArray(int[] array)
+        {
+            int sum = 0;
+            for (int i = 0; i < array.Length; i++)
+                sum += array[i];
+            
+            return 1d * sum / array.Length;
+        }
+
+        static void Main(string[] args)
+        {
+            int[] array1 = new int[10];
```

Comment:
I have an idea for this moment. So, what do you think about set array size in `InitializeArray` function?
i think that this way just fits under one of OOP principle such as `Single Responsibility`.
you can set array size in function or refer size by function parameter.

Reply:
Hmm, Do you mean that something like:
...
public void InitializeArray(int[] array, size )
{
Array.Resize(ref array, size);
...
}
public static void main ()
Program p = new Program()
int[] arrays;
byte size;
p.InitializeArray(arrays,size);
...

if so, is it correct to create it to resize it immediately?

Reply:
Or just an array should be created in the **InitializeArray** function which will return me an array with the specified size and initialized e.g

public int[] InitializeArray(byte sizeArray)
{
int[] array = new int[sizeArray] { 1, 2, ... n };
return array
}

Reply:
I mean, that you can use that `int[] array1 = new int[10];` in `InitializeArray`.
Or you can set array size from parameters like `public void InitializeArray(int[] array, size )`

Reply:
@EgorAntonovich I've fixed it. If it is not a good idea, please will inform me.